windows.md

```diff
@@ -0,0 +1,119 @@
+# Running/Developing OpenSphere on Windows 10+
+
+The following are instructions for getting OpenSphere building and running on a Windows 10 system using the new Windows Subsystem for Linux (WSL). Note All of the following instructions assume a Windows 10 build of 16215 or later. 
+
+## Installing WSL
+
+See the following instructions for setting up WSL on Windows 10. 
+
+https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+After installing the WSL feature and selecting a distribution, the following OpenSphere project prerequisites need to be installed using the WSL shell environment.
+
+	* [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+	* [NodeJS 8](https://nodejs.org/en/download/package-manager/)
+	* Oracle Java 8 SDK, OpenJDK should also work
+    * [Umbuntu](https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-get-on-ubuntu-16-04)
+    * [Open SUSE](https://en.opensuse.org/SDB:Installing_Java)
+	* [Yarn](https://yarnpkg.com/lang/en/docs/install/#linux-tab)
+
+After all prerequisites have been installed and verified, the OpenSphere getting started instructions should work.
+
+## Working with Windows
+
+The WSL environment has convenient mounts for all of the windows drives, so it easy to share files/folders with WSL and Windows by utilizing the drive mounts in WSL ```/mnt/<drive>/<path>```. For example, I configured the following path in Windows for the OpenSphere related projects:
+
+```c:\source\ngageoint```
+
+## Opening a WSL shell
+
+The windows WSL environment can be opened in a few ways, open a normal command prompt/PowerShell and just type wsl. Alternatively, find the distro or "wsl" from the start menu and run. This should open the WSL shell.
+
+## Cloning OpenSphere
+
+From within the (WSL) shell, cd to your working directory. Using the example, this would be cd ```/mnt/c/source/ngageoint```
+
+From this point forward, we can follow the Getting Started instructions provided by the OpenSphere project.
+	Fork/Clone the https://github.com/ngageoint/opensphere project in your project working directory

```

Comment:
This is inconsistent with the instructions that suggest checkout out `opensphere` into the `opensphere-yarn-workspace\workspace` directory.